mvpa2/generators/resampling.py

```diff
@@ -203,3 +203,71 @@ def generate(self, ds):
     def __str__(self):
         return _str(self, str(self._amount), n=self._attr, count=self.count,
                     apply_selection=self._apply_selection)
+
+
+class NonContiguous(Node):
+    """Generator to remove samples too close as measured with a sample attribute
+    eg.:
+    >>> partitioner = ChainNode([NFoldPartitioner(),
+    NonContiguous(attr='time',
+    dist=60,
+    partition_keep=2,
+    partition_trim=1)])
```

Comment:
please reformat using ReST formatting, otherwise would not look good on the web.  Checkout other docstrings for examples